src/Analyzer/CodeAnalyzer.cs

```diff
@@ -26,6 +26,11 @@ public class CodeAnalyzer : Codacy.Engine.Seed.CodeAnalyzer, IDisposable
         private readonly DiagnosticsRunner diagnosticsRunner;
 
         private readonly string tmpSonarLintFolder;
+        private static string[] blacklist = new string[] { "S1144", "S2325" };
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/gh/codacy/codacy-sonar-csharp/pullRequest?prid=5955346)